app/src/androidTest/java/com/github/se/project/ui/lesson/AddLessonTest.kt

```diff
@@ -171,22 +160,78 @@ class AddLessonTest {
     composeTestRule.onNodeWithTag("confirmButton").performClick()
     verify(navigationActions).navigateTo(anyString())
   }
-
+    */
```

Comment:
Was this test failing the CI? It was responsible for quite a bit of coverage, so it'd be nice if we found a way to make it work